src/app/states/layouts/page.mdx

```diff
@@ -0,0 +1,67 @@
+export const metadata = {
+  title: 'layouts',
+  description: 'An app state of component layouts. This section explains how to fetch the `layouts` state.',
+}
+
+# layouts
+
+An [app state](/guides/flux-architecture) of the layouts.
+You can obtain the `layouts` state and integrate it with React components.{{ className: 'lead' }}
+
+
+---
+
+## Data Structure
+A map of `[layout_name]: LayoutComponents` where:
+
+- `layout_name`: String
+- `LayoutComponents`: A class name of React component
+
+<Row>
+  <Col>
+    <Properties>
+      <Property name="main" type="full">
```

Comment:
```suggestion
      <Property name="main:full" type="Array<string>">
```
This should make it easier to understand.
The `:` is part of the layout name and therefore it should also be directly in the name